src/resource/page/ResourcePage.tsx

```diff
@@ -213,11 +224,11 @@ export default function ResourcePage({ id }: { id: string }): JSX.Element {
       {parentResources?.length > 0 && (
         <ResourceDetailsSection
           id="parent-resource-description"
-          title="Yläkohteet">
+          title="Toimipisteet">
```

Comment:
Think this will be ok as long as the depth of resource-subresource stays at 2. If I remember right api permits higher depth but in practice the depth is gonna stay at 2.

Reply:
Should we add unitResouce check here as well?